sematic/log_reader.py

```diff
@@ -192,7 +191,15 @@ def load_log_lines(
         max_lines,
         filter_strings,
     )
+    default_log_info_message = None
     run = object_source.get_run(run_id)
+
+    if run.original_run_id is not None:
+        # we source the logs from the original run, and also inform the user of this
```

Comment:
This switch reminds me that viewing the logs via the Grafana plugin is going to be pretty broken compared to using the built-in logging, as it won't be able to do things like this. It also won't be able to:
- read logs for inline runs
- read logs after the k8s pod is gone

Should we consider just deprecating the Grafana plugin at this point? There has been a case made that it will allow you to view stats (ex: CPU usage) as well, but those will suffer from the same deficiencies as the logs (breaks for inline, cloned runs, pods that are gone). Obviously this is not something to be addressed in this PR.

Reply:
I've actually never used that feature, and I expect users haven't either. We should however improve the observability / telemetry instead of just deprecating its current implementation.